Source/Analytics/Web/src/components/healthRisk/CaseReportByHealthRiskTable.js

```diff
@@ -44,46 +46,60 @@ class CaseReportByHealthRiskTable extends Component {
     }
 
     render() {
-        let styles={
-          marginBottom: 20
-        };
         return (
-          <div style={styles}>
-            <Typography variant="h5">No. of reports per health risk per time period</Typography>
-            <Paper>
-              <Table>
-                <TableHead>
+          <div className="tableContainer">
+            <h2 className="headline">Reports for the last 4 weeks</h2>
+              <Table className="table">
+                <TableHead className="tableHead">
                   <TableRow>
                     <TableCell></TableCell>
-                    <TableCell align="right">0-6 days</TableCell>
-                    <TableCell align="right">7-13 days</TableCell>
-                    <TableCell align="right">14-21 days</TableCell>
-                    <TableCell align="right">22-27 days</TableCell>
+                    <TableCell className="headerText">Reports last week (today - day 6)</TableCell>
+                    <TableCell className="headerText">Reports 1 week ago (day 7-13)</TableCell>
+                    <TableCell className="headerText">Reports 2 weeks ago (day 14-20)</TableCell>
+                    <TableCell className="headerText">Reports 3 weeks ago (day 21-27)</TableCell>
+                    <TableCell className="headerText">Total no. of reports (last 28 days)</TableCell>
                   </TableRow>
                 </TableHead>
                 <TableBody>
-                  {Object.keys(this.state.healthRisks).map(key => (
+                  {Object.keys(this.state.healthRisks).map(key => {
+                  var total = this.state.healthRisks[key].days0to6
+                              + this.state.healthRisks[key].days7to13
+                              + this.state.healthRisks[key].days14to20
+                              + this.state.healthRisks[key].days21to27;
+                  return (
                     <TableRow key={this.state.healthRisks[key].healthRiskName}>
-                      <TableCell component="th" scope="row">
+                      <TableCell className="cell">
                         {this.state.healthRisks[key].healthRiskName}
                       </TableCell>
-                      <TableCell align="right" style={this.state.healthRisks[key].days0to6 === 0 ? {color: "#B5B5B5"} : {}}>
+                      <TableCell 
+                        className="cell"
+                        style={this.state.healthRisks[key].days0to6 === 0 ? {color: "#B5B5B5"} : {}}>
```

Comment:
should this rather be set as a conditional class? ie we can have the classes cell and cell--empty. Where cell--empty overrides the color. Same for all TableCell in the rest of the files